intro2.md

```diff
@@ -0,0 +1,405 @@
+uses the same mechanism as L1 state sync, but must additionally include a
+signature from the sequencer to ensure the legitimacy of the data.
+
+## User Transactions and The Sequencer
+
+As a user, once you have [bridged] (\*) some ETH over to Optimistic Ethereum,
+you will probably interact with it though your wallet, such as Metamask. Add the
+network (chainID 10, [JSON-RPC] endpoint `https://mainnet.optimism.io/`) and
+you're good to go.
+
+(\*) L1 -> L2 deposits will be explained later.
+
+[bridged]: https://www.optimism.io/apps/bridges
+[JSON-RPC]: https://github.com/ethereum/execution-apis
+
+When you send a transaction, it is sent to the node called the sequencer. The
+sequencer will run verify your transaction, execute it if valid, and confirm it.
+This happens much faster than on Ethereum mainnet (around ~1s).
+
+There is currently a single sequencer operated by Optimism PBC. We expect future
+versions of this specification to introduce sequencer decentralization. However,
+despite the sequencer being decentralized, trust is not required. As we will
+see, the sequencer posts its results on Ethereum mainnet (henceforth: layer 1 or
+L1), where they can be permissionlessly challenged with a fraud proof. If the
+sequencer temporarily goes down, Optimism will remain live because users are
+able to submit L2 transactions on L1 (of course, this implies the loss of the
+increased throughput and lower fees).
+
+L2 transactions are identical to L1 transactions, except that they use the
+Optimistic L2 state instead of the L1 state. This includes account balances,
+deployed contracts and contract storage.
+
+## L2 Batches and Blocks
+
+Optimistic Ethereum has blocks, however these work slightly differently from L1
+blocks, which are mined by proof-of-work and will soon be decided by
+proof-of-stake.
+
+Optimistic Ethereum has two kind of blocks:
+1. L2 deposit blocks
+2. L2 sequencer blocks
+
+As it confirms transactions, the sequencer accumulates these transaction in a
+"batch". The transactions in such batches will become **_L2 sequencer block_**
+once posted to the Optimistic Ethereum contracts on L1. Please note that there
+is no simple mapping between batches and sequencer blocks: a batch may "contain"
+multiple blocks or even partial blocks. Batches are simply a means of grouping
+transactions together for submitting transactions to L1.
+
+The sequencer decides which L@ sequencer block a transaction belongs to,
+although this decision is constrained by the protocol. For instance, block
+numbers assigned to transactions increase monotonically. See the section [on the
+`NUMBER` opcode](#NUMBER) for more details.
+
+Just like batches, L1 blocks can contain multiple L2 blocks, or even partial
+blocks. They can also have received zero, one, or many batches from the
+sequencer.
+
+**_L2 deposit blocks_**, on the other hand, arise from L1 blocks. There is one
+L2 deposit block per L1 block. These deposit blocks are implicitly created by
```

Comment:
I think that it's better to conceptualize the deposit blocks as being "created" by L1, not by the sequencer. The reason is because the main purpose of deposit blocks is to provide a mechanism by which one can interact with the rollup *without* relying on the sequencer. This is such a hard concept to communicate 😭

Reply:
Just making a note to relate our discussion from today, where I make the point I'm not a fan of this framing, because the sequencer decides on L2 deposit block numbers. (The sequencer does however need to respect the rule, i.e. include the deposit block within its sequencing window.)

Reply:
This language is no longer present.